content/docs/use-cases/shared-development-server.md

```diff
@@ -46,18 +46,16 @@ $ sudo find /home/shared/dvc-cache -type f -exec chmod 0444 {} \;
 $ sudo chown -R myuser:ourgroup /home/shared/dvc-cache/
 ```
 
-## Configure the external shared cache
+## Configure the shared cache
 
-Tell DVC to use the directory we've set up above as the <abbr>cache</abbr> for
-your <abbr>project</abbr>:
+A <abbr>cache</abbr> outside the <abbr>workspace</abbr> is called an
```

Comment:
the `<abbr>cache</abbr>` contents (from https://github.com/iterative/dvc.org/blob/master/content/docs/user-guide/basic-concepts/dvc-cache.md) doesn't seem consistent with this para

Reply:
How so?

Reply:
the tooltip doesn't mention anything about the cache being external, but this para is abot externally located caches. Not sure if it may cause further confusion for people who have no idea what's going on.

Reply:
Gotcha. I think that the part "by default in `.dvc/cache`" suggests it can be located elsewhere. And nothing there limits it from being external.

I guess that's what @iesahin's https://github.com/iterative/dvc.org/pull/2475#discussion_r633674748 was too.